realm/realm-library/src/main/java/io/realm/internal/Group.java

```diff
@@ -226,6 +226,15 @@ public void writeToFile(File file, byte[] key) throws IOException {
         return nativeWriteToMem(nativePtr);
     }
 
+    /*
+     * Check if the Group contains any objects.
+     *
+     * @return {@code true} if empty, @{code false} otherwise.
+     */
+    public boolean isEmptyTables() {
```

Comment:
I don't know whether Group.size() and Group.isEmpty() are useless. Is it useless? Should I mark them as deprecated or just remove them?